lua/terrortown/menus/gamemode/equipment.lua

```diff
@@ -28,15 +28,14 @@ function CLGAMEMODEMENU:InitializeVirtualMenus()
 	for i = 1, #equipments do
 		local equipment = equipments[i]
 
-		-- Only keep ttt-equipments that are cached
-		if not equipment.ttt2_cached_material and not equipment.ttt2_cached_model then continue end
+		if not equipment.isEquipment then continue end
```

Comment:
Don't ask me, I only put this into a more sensible variable. But let me try to explain what the code does:

If an equipment entity is loaded, its material is cached. If the material is not the id material (the default used in the weapon base), then it is assumed to be a valid equipment weapon.
Previously this was done by checking if the material is cached. This caused problems because many weapons (pistols etc) do not have an icon. Therefore these were missing. I introduced this new variable that keeps the original implementation, while also caching the material for all weapons, no matter their type.

If this is a good approach is debatable. But I wanted to keep the old behavior. We can discuss this and change it in a future PR